### ISSUE TITLE
[RFC0031] Implement Cloud Native Buildpacks as system buildpacks

### ISSUE THREAD

**Issue by loewenstein-sap** [cloudfoundry/community#965]
This is a tracking issue for the implementation of https://github.com/cloudfoundry/community/blob/main/toc/rfc/rfc-0031-system-cnb.md

### PRs to be reviewed and merged
- [ ] https://github.com/cloudfoundry/cnbapplifecycle/pull/30
- [ ] https://github.com/cloudfoundry/cloud_controller_ng/pull/3898
- [ ] https://github.com/cloudfoundry/cli/pull/3497
- [ ] https://github.com/cloudfoundry/docs-buildpacks/pull/331
- [ ] https://github.com/cloudfoundry/cf-acceptance-tests/pull/1563

**Comment by jochenehret** [comment]
Yes, we have one environment where the "cnb" test suite is active:
https://concourse.wg-ard.ci.cloudfoundry.org/teams/main/pipelines/cf-deployment/jobs/fresh-cats/builds/2145
